webpack.config.js

```diff
@@ -141,6 +146,7 @@ module.exports = {
             },
           },
         ],
+        type: 'javascript/auto',
```

Comment:
This is supposed to be the workaround for continuing to use `file-loader`, but I'm not having any luck with it yet. I'm wondering if it's because we're using `file-loader` and `image-webpack-loader` in the same ruleset. I tried removing `image-webpack-loader` here, just to see if it would work since it looks like that's just providing optimizations, but no dice.